custom-layer-example/demo-composite-layer.js

```diff
@@ -0,0 +1,8 @@
+import { CompositeLayer } from "@deck.gl/core";
+import { ScatterplotLayer } from "@deck.gl/layers";
+
+export class DemoCompositeLayer extends CompositeLayer {
```

Comment:
any ideas on a more practical example are welcome

this is minimal but not necessarily useful